src/languages/javascript.js

```diff
@@ -272,6 +272,12 @@ export default function(hljs) {
           PARAMS
         ]
 
+      },
+      {
+        beginKeywords: "(setInterval|setTimeout)",
+        keywords: "setInterval setTimeout",
+        end: /\(/,
+        excludeEnd: true,
```

Comment:
Is there some reason these can't simply be added to built-ins in `keywords`?

Reply:
Oh yes I missed that